subsys/net/lib/nrf_cloud/include/nrf_cloud_agps_schema_v1.h

```diff
@@ -36,6 +36,7 @@ enum nrf_cloud_agps_type {
 	NRF_CLOUD_AGPS_GPS_SYSTEM_CLOCK			= 7,
 	NRF_CLOUD_AGPS_LOCATION				= 8,
 	NRF_CLOUD_AGPS_INTEGRITY			= 9,
+	NRF_CLOUD_AGPS_LAST				= NRF_CLOUD_AGPS_INTEGRITY
```

Comment:
i like to include an extra `_` just to indicate it isn't a proper enum value.  not sure if anyone else likes this method 😆 

`NRF_CLOUD_AGPS__LAST`

Reply:
I didn't write that code, but I can make the change.